x-pack/plugin/esql/compute/gen/src/main/java/org/elasticsearch/compute/gen/AggregatorProcessor.java

```diff
@@ -133,4 +143,23 @@ public static void write(Object origination, String what, JavaFile file, Process
             throw new RuntimeException(e);
         }
     }
+
+    private static List<TypeMirror> warnExceptions(Element aggregatorMethod) {
+        List<TypeMirror> result = new ArrayList<>();
+        for (var mirror : aggregatorMethod.getAnnotationMirrors()) {
+            String annotationType = mirror.getAnnotationType().toString();
+            if (annotationType.equals(Aggregator.class.getName()) || annotationType.equals(GroupingAggregator.class.getName())) {
+
+                for (var e : mirror.getElementValues().entrySet()) {
+                    if (false == e.getKey().getSimpleName().toString().equals("warnExceptions")) {
+                        continue;
+                    }
+                    for (var v : (List<?>) e.getValue().getValue()) {
+                        result.add((TypeMirror) ((AnnotationValue) v).getValue());
+                    }
+                }
+            }
+        }
+        return result;
```

Comment:
I think this is a copy of some other code. Could we make a static reference instead here? At least for some of it. It's so weird.